pycloudlib/instance.py

```diff
@@ -485,19 +485,25 @@ def _tmpfile(self):
         self._tmp_count += 1
         return path
 
-    def _wait_for_execute(self, old_boot_id=None):
-        """Wait until we can execute a command in the instance.
+    def _wait_for_execute(self, old_boot_id=None, timeout: int = 40):
+        """
+        Wait until we can execute a command in the instance.
+
+        Args:
+            old_boot_id: boot id before restart. If provided, we will wait
+            until we find a new boot id
+            timeout: time to wait for instance to be reachable in minutes
 
-        If old_boot_id is specified, we use its value to wait until we
```

Comment:
I think this sentence is still relevant, could we please keep it?

Reply:
I kept this information, just reworded above in the new docstring. Is the rewording poor/is detail lost?